### PR TITLE
Feat/cursor

### PR DESCRIPTION
**Pull request by aureldent**
Make cursor a class parameters an allow to set it in get_call and iter_call
Add/update some tests related to that and rework iter_call and get_call.
This is usefull when wanting to retry calls.

### CODE REVIEW

File: lumapps/api/client.py [hunk 1]
```diff
@@ -356,7 +369,7 @@ def upload_call(self, fpath: Path, metadata: dict, *name_parts, **params):
         resp.raise_for_status()
         return resp.json()
 
-    def get_call(self, *name_parts, **params):
+    def get_call(self, *name_parts, cursor=None, **params):
```
Comment (ulodciv) [inline lumapps/api/client.py:372]: Please make this happen using what already exists, ie parameters or payloads.

File: lumapps/api/client.py [hunk 2]
```diff
@@ -376,29 +389,42 @@ def get_call(self, *name_parts, **params):
         """
         name_parts = _parse_endpoint_parts(name_parts)
         items = []
-        cursor = None
+        self.cursor = cursor
```
Comment (ulodciv) [inline lumapps/api/client.py:392]: Why must this become an object-level variable?

Comment (aureldent) [reply lumapps/api/client.py:392]: To be able to get it after a call failed for instance.

File: lumapps/api/client.py (hunk 1 above)
Comment (aureldent) [reply lumapps/api/client.py]: You're right, done it.

**Pull request merged**